src/paginator.js

```diff
@@ -0,0 +1,300 @@
+import assign from 'lodash.assign';
+import omit from 'lodash.omit';
+
+
+export const styles = Object.freeze({
+  pageNumber: 1,
```

Comment:
also, `pageNumber` is a misleading label - especially when the value is an actual number